openstack/barbican/templates/api-deployment.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     type: api
     component: barbican
 spec:
-  replicas: {{ .Values.pod.replicas.api }}
+  replicas: {{ .Values.api.replicas }}
```

Comment:
I suggest to set a default

Reply:
we cannot set to default due CPU constraints in qa-de-2, 

example: https://github.wdf.sap.corp/cc/secrets/blob/barbican_qa-de-2_values/qa-de-2/values/barbican.yaml#L44

Reply:
That is a different perspective - you can override the value. If you don't want a default - at least mark the value as required please

Reply:
https://github.com/sapcc/helm-charts/pull/1238/commits/efb62f288c1eb7820187c484ecd4e1891a30bb33 hope this is fine?

rabbitmq_notifications which is required for barbican-audit which is enabled in all regions expect staging